python/foglamp/services/core/server.py

```diff
@@ -136,6 +139,11 @@ class Server:
                            'for the interface',
             'type': 'JSON',
             'default': '{"providers": ["username", "ldap"] }'
+        },
+        'certificateName': {
+            'description': 'Certificate file name',
+            'type': 'string',
+            'default': 'foglamp'
```

Comment:
@MarkRiddoch Please confirm `certificateName` key is okay.

Reply:
Yes, fine with that - I will add to admin API document